MapboxNavigation/NavigationViewController.swift

```diff
@@ -123,15 +129,15 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
     }
     
     /** 
-     `destination` is an instance of `MGLAnnotation` that will be showned on
+     `destination` is an instance of `MGLAnnotation` that will be shown on
      on the destination of your route. The last coordinate of the route will be
      used if no destination is given.
     */
     public var destination: MGLAnnotation!
     
     /**
      `directions` is an instance of `Directions` need for rerouting.
-     See [MapboxDirections.swift](https://github.com/mapbox/MapboxDirections.swift)
+     See [Mapbox Directions](https://mapbox.github.io/mapbox-navigation-ios/directions/)
```

Comment:
Nit: Mapbox Directions is the name of the [server-side product](https://www.mapbox.com/directions/); for now, the client-side project is known as MapboxDirections.swift or MapboxDirections. (I know…) I’d love to sort this out in the near future, but avoiding name proliferation would be helpful in the meantime.